modules/data.atmosphere/NEWS.md

```diff
@@ -0,0 +1,36 @@
+# PEcAn.data.atmosphere 1.7.2.9000
+
+## Fixed
+
+* Extensive code cleanup and fixes to function documentation [@moki1202, ; @meetagrawal, #3212; @aariq, #3055; @Its-Maniaco, #2954]
+* Retired packages `maptools` and `rgdal` are no longer imported [@istfer, #3228]
+* Unit conversion now uses `PEcAn.utils::ud_convert` to avoid relying on orphaned package `udunits2` [@nanu1695, #2989]
+* `download.MERRA()` now respects `overwrite = FALSE` [@meetagrawal09, #3169]
+* `download.MERRA()` can now find data from September of 2020 [@KristinaRiemer, #2888]
+* `download.Fluxnet2015()` now gives a more useful message when passed a malformed URL [@meetagrawal, #3130]
+
+## Added
+
+* Newly supported met data sources:
+	- ERA5 via `ERA5_met_process()` [@DongchenZ, #2886]
+	- NOAA GEFS via `download_NOAA_GEFS_EFI()` [@helge22a, #3174]
+		(replaces NOAA_GEFS_downscale?)
```

Comment:
No, doesn't replace. The EFI version can only grab files for the subset of sites on data.ecoforecast.org, which at the moment is just NEON. We still need to maintain the original GEFS download functions for all other sites